world-client/src/features/chat.js

```diff
@@ -68,7 +68,24 @@ export default class Chat extends Feature {
         li.appendChild(text);
         // Finally add message
         messages.appendChild(li);
+        
+
+        var player = wop.state.getPlayers().find((x) => x.id === message.id);
+        if (!player)
+          break;
+        player.currentMessage = message.text;
+        if (player.chatTimeoutHandle > -1) {
+          clearTimeout(player.chatTimeoutHandle || null);
+        }
```

Comment:
This clearTimeout call is unnecessary as it won't do anything. setTimeout function only executes once and Javascript doesn't need manual garbage collection.